src/core/qgsapplication.cpp

```diff
@@ -556,6 +556,13 @@ QCursor QgsApplication::getThemeCursor( Cursor cursor )
   {
     // Apply scaling
     float scale = Qgis::UI_SCALE_FACTOR * app->fontMetrics().height() / 32.0;
+#ifdef Q_OS_MACX
```

Comment:
should this always apply if the pixel ratio >= 2? I can't test, but I'm not sure why this would be osx specific

Reply:
AFAIK you can only have a pixel ratio of 1 or 2 on osx at the moment, and apple has standardised this on purpose. 2 is the default when a retina display is used and 1 for non-retina display or when you set the low resolution flag for the application.

Reply:
What I mean is that potentially other platforms (non OSX) could have a > 1 pixel ratio - and should probably also get the fix

Reply:
Yip my comment was in relation to keeping this mac specific.